Detectors/FIT/FT0/workflow/include/FT0Workflow/FT0DataReaderDPLSpec.h

```diff
@@ -40,13 +40,26 @@ class FT0DataReaderDPLSpec : public Task
   FT0DataReaderDPLSpec(const RawReader& rawReader) : mRawReader(rawReader) {}
   FT0DataReaderDPLSpec() = default;
   ~FT0DataReaderDPLSpec() override = default;
+  typedef RawReader RawReader_t;
```

Comment:
Why? This creates a more complicated alias name for the RawReader, and both names are accessible from within this class anyway... If there's a name ambiguity, wouldn't it be better to simply change the name in line 36 from RawReader to RawReader_t ?

Reply:
RawReader could be needed as member type of FT0DataReaderDPLSpec. This is good practice to save template arguments.